src/main/java/managers/JSExecutor.java

```diff
@@ -15,6 +16,24 @@ public JSExecutor(WebDriver driver) {
         js = (JavascriptExecutor) driver;
     }
 
+    public boolean waitForJQueryToLoad() {
+        ExpectedCondition<Boolean> jQueryLoaded = (WebDriver driver) ->
+                (Long) executeScript("return jQuery.active") == 0;
+
+        try {
+            return Context.driverManager.wait.until(jQueryLoaded);
+        }catch (Exception e){
+            // no jQuery present
+            return true;
+        }
+    }
+
+    public boolean waitForJSToLoad() {
```

Comment:
similar method is in DriverManager class, waitForPageToLoad() - remove 1 to avoid duplicates

Reply:
one from DriverManager was removed